src/poliastro/tests/tests_plotting/test_static.py

```diff
@@ -116,3 +117,12 @@ def test_redraw_keeps_trajectories():
     op.set_frame(*mars.pqw())
 
     assert len(op.trajectories) == 2
+
+
+@pytest.mark.mpl_image_compare
+def test_trail_plotting():
```

Comment:
I think this test is not testing anything. Tried adding a red thick line in the image and even removing it, and still passed.